solr/core/src/java/org/apache/solr/schema/LatLonPointSpatialField.java

```diff
@@ -75,8 +77,16 @@ protected SpatialStrategy newSpatialStrategy(String fieldName) {
     return new LatLonPointSpatialStrategy(ctx, fieldName, schemaField.indexed(), schemaField.hasDocValues());
   }
   
-  public String geoValueToStringValue(long value) {
-    return new String(decodeLatitudeCeil(value) + "," + decodeLongitudeCeil(value));
+  /**
+   * Converts to "lat, lon"
+   * @param value Non-null; stored location field data
+   * @return Non-null; "lat, lon" with 6 decimal point precision
```

Comment:
Why 6 decimal points?  Is that sufficient to represent the data to as much precision as is decoded?  Perhaps instead of putting the constant '6' in the code, it should be calculated so that we can see how 6 is arrived at.  What does that translate to in the metric system?

Reply:
6 was just based on what i read, For 40.299599,-74.082728 it is 40°17'58.52",74°4'57.79".  I will remove it

Reply:
By "What does that translate to in the metric system?" I mean for example how many meters would this translate to with 6 decimal points?

"6 was just based on what i read"   read where?

Reply:
Based on the article it would be close 111mm
https://gis.stackexchange.com/a/8674
https://en.wikipedia.org/wiki/Decimal_degrees